packages/validator/README.md

```diff
@@ -18,7 +18,7 @@ Where array `messages` contains message objects:
     message: string
 }
 ```
-If provided OIS is valid, parameter `valid` will be true, however parameter `messages` still might contain messages with `level` set to `warning`. If `valid` is `false`, there will be always one or more error messages. Similarly functions `isApiSpecsValid` and `isEndpointsValid` check only the API and endpoint specifications contained in the OIS.
+If provided OIS is valid, parameter `valid` will be true, however parameter `messages` still might contain messages with `level` set to `warning`. If `valid` is `false`, there will be always one or more error messages. Similarly functions `isApiSpecsValid` and `isEndpointsValid` check only the API and endpoint specifications contained in the OIS, as well as `isConfigSecurityValid` validates `config.json` and `security.json`
 
 The validator can be run as an NPM script, by providing the paths to the JSON file that will be checked and the JSON file to use as the template (validator specification structure). You can run the validator as script using the following command: `npm run validate --specs="[specsFile]" --template="[templateFile]"`
```

Comment:
I think it makes more sense for the command to go `npm run validate <specs> <template>` because "validate according to `<template>`" is the action and `<specs>` is the argument

Reply:
A copy-pasteable command that uses an example `specs` file would be useful here. Because the first thing I do after reading this first paragraph is to go out and look for example files to try this out.

Reply:
I don't quite follow "the command to go `npm run validate <specs> <template>`", currently you can launch the command exactly as you say, did you mean `npm run validate <template> <specs>`?

Example added in 943f6df

Reply:
I was just saying that it would make more sense if the order of the arguments were the other way around